app/workers/article_worker.rb

```diff
@@ -0,0 +1,14 @@
+# A background worker that builds new Articles from links
+# or updates existing articles with excerpts if they
+# have been created by the ui
+class ArticleWorker
+  include Sidekiq::Worker
+
+  def perform(url, article_id = nil)
+    uri = URI(url)
+    if uri.host =~ /nytimes.com/
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.